src/components/OSCALLoader.js

```diff
@@ -69,7 +69,7 @@ export default function OSCALLoader(props) {
           if (!response.ok) throw new Error(response.status);
           else return response.json();
         },
-        (err) => handleFetchError(err)
+        (err) => handleRestError(err)
```

Comment:
Realizing now that we actually can probably just do

```suggestion
        handleRestError
```

here. Honestly not sure whether that's cleaner.